File: Programacao  2/Phyton/Ex27.py
Num = int(input('Digite um numero positivo para a tabuada: '))

while Num <=0:
    print(f"Erro! Apenas numeros positivos")
    Num = int(input("Digite novamente: "))

a = int(input('Digite o início do intervalo da tabuada: '))
b = int(input('Digite o fim do intervalo da tabuada: '))

while b<a:
    print(f"Erro! Fim do intervalo deve ser maior que o início")
    b = int(input("Digite novamente o fim do intervalo: "))

for i in range(b, a-1, -1): 
    Res = Num * i
    print(f'{Num} x {i} = {Res}')

File: Programacao  2/C/Ex27.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{

int num, A, B, res;

                    
printf("\nDigite um numero positivo para a tabuada: ");
scanf("%i",&num);

while(num<=0){
printf("Erro!\nApenas numeros positivos. Digite novamente: ");
scanf("%i",&num);
}

printf("\nDigite o inicio do intervalo da tabuada: ");
scanf("%i",&A);

printf("\nDigite o fim do intervalo da tabuada: ");
scanf("%i",&B);

while(B<A){
printf("Erro!\nFim do Intervalo deve ser maior que o inicio. Digite novamente o fim do intervalo: ");
scanf("%i",&B);    
}


for (int i=B; i>=A; i=i-1) {
res = num * i; 
printf("%i x %i = %i\n", num, i, res);
}
return 0;

}